src/components/SplashScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import Circle from "./splashScreen/Circle";
+import "./splashScreen/style/SplashScreen.css";
```

Comment:
**nitpick:** We should use CSS modules here, as now you are using global classNames, which can lead to styles overwrite. When you use CSS modules, you isolate classnames only to the specific component, so you don't have to use BEM then. Example:

```tsx
import styles from "./SplashScreen.module.css";

export default function SplashScreen(props: SplashScreenProps) {
  return (
    <div className={styles.splashScreen}>
```